includes/derivatives.inc

```diff
@@ -151,7 +152,18 @@ function islandora_pdf_add_pdfa_derivative(AbstractObject $object, $force = FALS
   $executable = variable_get('islandora_pdf_path_to_gs', '/usr/bin/gs');
   $temp = file_create_filename('pdfa.pdf', 'temporary://');
   $derivative_file_uri = drupal_realpath($temp);
-  $command = "$executable -dPDFA -dNOOUTERSAVE -dUseCIEColor -sProcessColorModel=DeviceRGB -sDEVICE=pdfwrite -o $derivative_file_uri -dPDFACompatibilityPolicy=1 $source ";
+
+  // XXX. As of Ghostscript version 9.11, the use of -dUseCIEColor is not
+  // recommended.
+  // see https://ghostscript.com/pipermail/gs-devel/2014-July/009693.html.
+  $use_ci_color = "-dUseCIEColor";
+  $version_command = "$executable --version";
+  exec($version_command, $version);
```

Comment:
How much time does this add to the derivative creation process to run this version command every time? I think maybe we should do some performance testing on this. If it adds significant time, maybe we could have it as an admin option, or cache the result of this for some time?

Reply:
Using it as an admin option seems to a good bet

Reply:
Should probably have an update hook to set the variable to the correct value.

Reply:
Thanks for pointing that out! Adding in the .install file